### ISSUE TITLE
Immense stuttering after playing for a while

### ISSUE THREAD

**Issue by Joshua-Ashton** [labwc/labwc#495]
Only affects XWayland apps

After playing a game etc for a while, everything now starts to run reeeally slowly on master.

This is reproducible by just playing a game on Proton or a native Linux game for a while. And even spreads to running things like glxgears.

The stutter only happens when pressing a key or moving the mouse. Did we change anything input related recently?

**Comment by johanmalm** [comment]
There have been quite a few changes relating to xwayland clients positioning in the last couple of months.
It might relate to refactoring the `configure_requests` or a `wlr_scene_node_set_position()` in the wrong place or similar.

If you've got time to checkout some commits to check where in the history the stuttering starts, I'd try the following:

```
52d1e32 view: view_move() s/double/int/ for x and y
065c37d xwayland: Process move-only client configure requests more quickly
f1ba0a8 src/view.c: Prevent crash by killing a moving application
bfff9d0 xwayland.c: fix position bug
80792d4 (Partly) fix handling of client-initiated configure requests
```

(by `git log --oneline src/xwayland.c | head -n 5`)

and...

```
ac5c9da src/xwayland-unmanaged.c: Prevent racing configure event to reposition
5d0faa2 xwayland: Process unmanaged client configure requests more quickly
bfff9d0 xwayland.c: fix position bug
80792d4 (Partly) fix handling of client-initiated configure requests
```

(by `git log --oneline src/xwayland-unmanaged.c | head -n 4`)

**Comment by Consolatis** [comment]
Might be the same reason as for the now (hopefully) fixed
- #512

Does this still happen on latest master?

**Comment by Consolatis** [comment]
Assuming this is fixed on 0.6 / latest master. Shout if this still happens and we'll reopen.
